src/pods/canvas/canvas.business.ts

```diff
@@ -73,3 +79,75 @@ export const calculateRelationXCoordinate = (
   xOrigin: calculateRelationXCoordinateOrigin(tableOrigin, tableDestination),
   xDestination: calculateRelationXCoordinateEnd(tableOrigin, tableDestination),
 });
+
+const seekField = (
+  fieldId: GUID,
+  YPosition: number,
+  fields: FieldVm[]
+): seekResult => {
+  let found = false;
+  let parentCollapsed = false;
+  let newYPosition = YPosition;
+
+  for (let i = 0; i < fields.length && !found; i++) {
+    const field = fields[i];
+    if (field.id === fieldId) {
+      found = true;
+      return {
+        found,
+        parentCollapsed,
+        YPosition: newYPosition,
+      };
+    } else {
+      if (!parentCollapsed) {
+        newYPosition += ROW_HEIGHT;
+      }
+
+      if (!parentCollapsed && field.isCollapsed) {
+        parentCollapsed = true;
+      }
+
+      if (
+        field.type === 'object' &&
+        field.children &&
+        field.children.length > 0
+      ) {
+        const result = seekField(fieldId, newYPosition, field.children);
+        found = result.found;
+        newYPosition = parentCollapsed ? newYPosition : result.YPosition;
+        parentCollapsed = result.parentCollapsed;
+
+        if (found) {
+          return {
+            found,
+            parentCollapsed,
+            YPosition: newYPosition,
+          };
```

Comment:
Here we could make use of `buildFieldFoundResponse ` that we created before

Reply:
It's fixed